### DISCUSSION TITLE
Comparison of Regions-Gene linkages from SCENIC+ vs ABC model

### DISCUSSION THREAD

**Discussion by Xieeeee** [aertslab/scenicplus#513] [Q&A]
Hi,
Thanks for this great tool! I just wonder whether you have done a comparison of the overlap between long range interactions calculated with Hi-C data (ABC model) vs the regions-gene linkage from scenic+? I found in the [manuscript](https://www.nature.com/articles/s41592-023-01938-4/figures/3) only the correlation of Hi-C across top100 genes.

The reason I asked is bc our lab is having the single cell multiome, Hi-C and H3K27ac data right now and we are deciding how to reconcile the differences between these two strategies. Any comments or input would be helpful.

Thanks,
Yang

**Comment by SeppeDeWinter** [comment]
Hi @Xieeeee 

No, we have not done that comparison I'm afraid. We only did the comparisons shown in the manuscript.

Sorry I can not help you further.

All the best,

Seppe
